tests/contracts/root_chain/test_process_exits.py

```diff
@@ -279,6 +279,24 @@ def test_finalize_challenged_exit_will_not_send_funds(testlang):
     assert post_balance == pre_balance
 
 
+def test_finalize_challenged_exit_does_not_emit_events(testlang):
+    owner, finalizer, amount = testlang.accounts[0], testlang.accounts[0], 100
+    deposit_id = testlang.deposit(owner, amount)
+    spend_id = testlang.spend_utxo([deposit_id], [owner.key], [(owner.address, NULL_ADDRESS, 100)])
+
+    testlang.start_standard_exit(spend_id, owner.key)
+    doublespend_id = testlang.spend_utxo([spend_id], [owner.key], [(owner.address, NULL_ADDRESS, 100)])
+
+    testlang.challenge_standard_exit(spend_id, doublespend_id)
+    assert testlang.get_standard_exit(spend_id) == [NULL_ADDRESS_HEX, NULL_ADDRESS_HEX, 0]
```

Comment:
Minor: I personally would try to only assert what I want to test. But not really a blocker of this merge.